Makefile

```
@@ -0,0 +1,15 @@
+CC ?= gcc
+PREFIX ?= /usr/local
+CFLAGS += -g
+
+all: dollarskip
+
+install: dollarskip
+	install -d $(DESTDIR)$(PREFIX)/bin
+	install -m 755 dollarskip $(DESTDIR)$(PREFIX)/bin/\$
+
+uninstall:
+	rm -f $(DESTDIR)$(PREFIX)/bin/\$
+
+clean:
+	rm -f dollarskip
```

Why the force remove?

I'd rather not use a force remove. Same applies above.

---

Force.

It only works here to prevent errors. This is the way makefiles are written to do this, since `-f` is the correct flag in this instance.

---

Ok, personally I don't like the sound of it. However for keeping simplicity, we'll use -f

---

we can also do this:
```suggestion
	-rm dollarskip
```
that tells `make` to ignore the error but it still reports it. personally I think using `-f` is better.